GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
```

Comment:
Language is tricky for these, so I'm going to be super picky. As far as glossary definitions are concerned:

- Roles are **named collections of permissions**.
- Roles may be **granted** to **users** or **groups**.

I know that's super picky.

There are a lot of reasons why people might want to define roles, but I think should avoid talking about them those reasons in the glossary. We should definitely talk about those sorts of things in a best practices document for Roles at some point, though.